.github/workflows/conference.yml

```diff
@@ -53,3 +53,21 @@ jobs:
           labels: |
             report
             automated pr
+  add_comment:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Add comment to the issue
+      uses: actions/github-script@v6
+      if: ${{ github.event.issue.user.type != 'User' }} #adds comment only if the issue is created by a bot
```

Comment:
We may need to test this and create a quick fix to see if it will work.

Reply:
Okay so how do i create an issue automatically to test it

Reply:
@JohananOppongAmoateng - I think there is an issue in how I wrote this that is going to require some updates.

This isn't your fault at all and purely mine.

Look at - https://github.com/BlackPythonDevs/blackpythondevs.github.io/issues/432 as an example

- The issue isn't actually created by a bot but is created by a user
- The issue could be discovered by using the template
  - the title starts with `[CONFERENCE]` and has the label `conference`
  
 I think we should create an issue that updates our request with this information and get a PR that fixes the existing action

> SIDENOTE: I missed that pre-commit failed on the run so we can update that file fixing the pre-commit changes as well.

Reply:
So a user creates the issue 
Check if conference issue
Fix pre-commit fail